### PR TITLE
feat: Optionally skip create table with argument to newAdapter

### LINKED ISSUE #418: Using a readonly data source: Can you skip creation of the data source?
Hi Casbin community,

My company and I are using casbin in a service-oriented architecture where there are multiple applications reading the casbin_rule table on an RDS Aurora cluster in AWS (MySQL-compatible). We have one permissions API that is responsible for writing the policies to the casbin_rule table, and ideally we would allow other services to have a read only connection to the database cluster for reading the casbin policies. We use the https://www.npmjs.com/package/casbin-sequelize-adapter. The reason for wanting to use a read only connection for services not responsible for writing the data, is so that we can scale out horizontally with a loadbalanced cluster of read replica databases. This would be a huge performance gain when scaling enterprise software. However we are not able to do this because the sequelize adapter calls `CasbinRule.sync()` on `open()` which in turn runs `CREATE TABLE IF NOT EXISTS` every time we try and read the table. Not only does this error out on a read only connection in SQL, this also seems excessive to check this every single time. We handle creating our database and table via a migration outside of the application itself. 

Looking through the adapter code as well as other casbin adapters, it seems that this is a casbin pattern; and not necessarily a SQL adapter pattern. Is there a casbin way of not creating the data source every time we connect to the database? I was thinking of forking the sequelize adapter code and making a configuration option, but was curious if there was already a plan or pattern in place that I should reference.

Thanks in advance for any help!

### PR DESCRIPTION
**Pull request by rogersgt**
Fix: https://github.com/casbin/node-casbin/issues/418

See my question int he issues section of the `node-casbin` repo: https://github.com/casbin/node-casbin/issues/418. If there is already a way to do this, or a better pattern, I'm all for it. But I really think this is needed for many enterprise implementations.

### CODE REVIEW

File: src/adapter.ts [hunk 1]
```diff
@@ -29,9 +29,11 @@ export class SequelizeAdapter implements Adapter {
   private readonly option: SequelizeAdapterOptions;
   private sequelize: Sequelize;
   private filtered = false;
+  private autoCreate = true;
 
-  constructor(option: SequelizeAdapterOptions) {
+  constructor(option: SequelizeAdapterOptions, autoCreate = true) {
```
Comment (hsluoyz) [inline src/adapter.ts:34]: autoCreateTable

File: src/adapter.ts [hunk 2]
```diff
@@ -47,9 +49,10 @@ export class SequelizeAdapter implements Adapter {
    * @param option sequelize connection option
    */
   public static async newAdapter(
```
Comment (hsluoyz) [inline src/adapter.ts:51]: Add the new usage to README.md

https://github.com/node-casbin/sequelize-adapter#simple-example

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Comment by hsluoyz** [comment]
@rogersgt

**Comment by hsluoyz** [comment]
@rogersgt hi, can you fix the issues according to the review?

**Comment by hsluoyz** [comment]
Replaced by: https://github.com/node-casbin/sequelize-adapter/pull/73

**Comment by rogersgt** [comment]
Sorry for the delay here. Thanks for making the change @terry-xuan-gao!

**Pull request closed without merging**